src/Components/Popover/PopoverDismissButton.tsx

```diff
@@ -0,0 +1,57 @@
+// Libraries
+import React, {Component} from 'react'
+import classnames from 'classnames'
+
+// Components
+import {SquareButton} from '../Button/Composed/SquareButton'
+
+// Styles
+import './Popover.scss'
+
+// Types
+import {
+  ComponentColor,
+  StandardProps,
+  IconFont,
+  ComponentSize,
+} from '../../Types'
+
+interface Props extends StandardProps {
+  /** Button color */
+  color: ComponentColor
+  /** Pass the "onHide" function in here */
+  onClick: any
```

Comment:
When I typed this as `() => void` I got some weird type errors that I could not figure out how to resolve:

```
TS2322: Type '(() => void) | undefined' is not assignable to type '() => void'.
  Type 'undefined' is not assignable to type '() => void'.
```

Reply:
You'd need to type it as `() => void) | undefined`